pkg/vf/virtio.go

```diff
@@ -230,39 +231,64 @@ func unixFd(fd uintptr) int {
 // https://developer.apple.com/documentation/virtualization/running_linux_in_a_virtual_machine#3880009
 func setRawMode(f *os.File) error {
 	// Get settings for terminal
-	attr, _ := unix.IoctlGetTermios(unixFd(f.Fd()), unix.TIOCGETA)
+	var attr unix.Termios
+	err := termios.Tcgetattr(f.Fd(), &attr)
```

Comment:
if and only if you repush, because attr is being used as a pointer, this can be the `if err :=` syntax (i believe)